sycl/test/aie/mandelbrot_hetero.cpp

```diff
@@ -0,0 +1,57 @@
+/// This test requires a display
+// REQUIRES: aie && false
+
+// RUN: %aie_clang %s -o %t.bin
+// RUN: %if_run_on_device %run_on_device %t.bin > %t.check 2>&1
+// RUN: %if_run_on_device FileCheck %s --input-file=%t.check
+
+#include "aie.hpp"
+#include "ext/graphics.hpp"
+#include <complex>
+
+int main(int argc, char **argv) {
+  aie::ext::application<uint8_t> a;
+  aie::device<8, 8> dev;
+  aie::queue q(dev);
+  /// The current maximum size of a memory module is 8192 bytes
+  /// sqrt(8192) ~ 90.5... so 90 is the largest integral value we can put here.
+  /// at least until the 8192 bytes goes away.
+  static auto constexpr image_size = 90;
```

Comment:
```suggestion
  auto constexpr image_size = 90;
```
I do not remember why I had all these `static`

Reply:
Now I remember. There are some cases where it was classes, so `static` member where required, but this is not the case here.